addons/editor-number-arrow-keys/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Arrow key incrementation in editor",
-  "description": "Use the up and down arrow keys to increase or decrease number values in block inputs.",
+  "description": "Use the up and down arrow keys to increase or decrease number values in numeric input fields.",
```

Comment:
Maybe we could add examples:

`numeric input fields, such as the input in the "wait () seconds" block, or the size in the sprite properties.`